lib/rucio/web/ui/static/rucio.js

```diff
@@ -1391,6 +1392,43 @@ RucioClient.prototype.delete_rse_attribute = function(options) {
     });
 };
 
+/* send trace */
+RucioClient.prototype.send_trace = function(options) {
+    check_token();
+    var url = this.url + '/traces/';
+    if (options.async == null) { options.async = true; }
+    jQuery.ajax({
+        url: url,
+        async: options.async,
+        crossDomain: true,
+        headers: this.headers,
```

Comment:
See above. The token and account are not needed, so you don't have to send them in the header.